iceoryx_hoofs/include/iceoryx_hoofs/error_handling/error_handling.hpp

```diff
@@ -17,20 +17,23 @@
 #ifndef IOX_HOOFS_ERROR_HANDLING_ERROR_HANDLING_HPP
 #define IOX_HOOFS_ERROR_HANDLING_ERROR_HANDLING_HPP
 
+/// @todo #590 rename this file to error_handler.hpp
```

Comment:
Done in #1028

Reply:
This is hard. Furthermore #1028  states the following: ` Separate errors of posh and C binding` and as it seems currently the include guards, include headers, cmake files are not yet adjusted there. Please create a separate PR for this the other one is already huge.

Reply:
On the 2nd thought, I think what we need to do is separate the `ErrorHandler` from `error_handling.hpp`. `iceoryx_hoofs`'s `error_handling.hpp` will have only one entry `EXPECTS_ENSURES_FAILED`. Added to #1099